src/auro-datepicker.js

```diff
@@ -695,7 +707,8 @@ export class AuroDatePicker extends LitElement {
               setCustomValidityRangeOverflow="${this.setCustomValidityRangeOverflow}"
               setCustomValidityRangeUnderflow="${this.setCustomValidityRangeUnderflow}"
               ?disabled="${this.disabled}"
-              .type="${this.type}">
+              .type="${this.type}"
+              part="datepickerInput">
```

Comment:
can this just be `input`? Also, doesn't look like you are handing range support with two inputs here.